src/entities/entries.js

```diff
@@ -53,7 +54,7 @@ export const processEntryFilters = (filters, framework, geoOptions) => {
     const result = [];
     Object.keys(filters).forEach((filterKey) => {
         const filterOptions = filters[filterKey];
-        if (!filterOptions) {
+        if (!isDefined(filterOptions)) {
```

Comment:
Use `isNotDefined` instead of `isDefined`. There are some edge-cases where isNotDefined and !isDefined have different result